boms/cloud-oss-bom/pom.xml

```diff
@@ -79,36 +79,16 @@
         <artifactId>grpc-okhttp</artifactId>
         <version>${io.grpc.version}</version>
       </dependency>
-      <dependency>
-        <groupId>io.grpc</groupId>
-        <artifactId>grpc-netty-shaded</artifactId>
-        <version>${io.grpc.version}</version>
-      </dependency>
       <dependency>
         <groupId>io.grpc</groupId>
         <artifactId>grpc-protobuf</artifactId>
         <version>${io.grpc.version}</version>
       </dependency>
-      <dependency>
-        <groupId>io.grpc</groupId>
-        <artifactId>grpc-stub</artifactId>
-        <version>${io.grpc.version}</version>
-      </dependency>
-      <dependency>
-        <groupId>io.grpc</groupId>
-        <artifactId>grpc-core</artifactId>
```

Comment:
Why are these modules being removed?

Reply:
They aren't. They were dupes and Maven complained.

Reply:
Could you alphabetize the list so that it's easy to see dupes?

Reply:
That raises the question of whether the order of dependencies in a dependencyManagement section matters in dependency resolution. I ask because the order of dependencies in a dependency section does matter. So far, it's just the order I pasted them in, but I wonder if we need to be more careful about what comes first?

Reply:
Excellent question... are you interested in doing the research?

Reply:
Looking over the maven docs, it doesn't seem to matter in any way I can see. I've uploaded a new version that alphabetizes the gRPC dependencies.